apps/token-manager/app/src/screens/Holders.js

```diff
@@ -224,4 +207,31 @@ function EntryActions({
   )
 }
 
+function TokenAmount({ balance, tokenDecimals, vestings }) {
```

Comment:
Moved the things above directly into this component; left it on the same file as it's related to the DataView only, but if preferred, we can move it.